pyproject.toml

```diff
@@ -16,6 +16,7 @@ alembic = "^1.7.5"
 ipython = {version = "^7.29.0", optional = true}
 psycopg2 = {version = "^2.9.2", optional = true}
 celery = {version = "^5.2.1", extras = ["redis"]}
+httpx = {version = "^0.18.2", optional = true}
```

Comment:
This would be used further down to add a new `legacy` extra, e.g.:

```
[tool.poetry.extras]
interactive = ["ipython"]
postgresql = ["psycopg2"]
legacy = ["httpx"]
```